dependencies.gradle

```diff
@@ -1,8 +1,8 @@
 ext {
-  androidPlugin = 'com.android.tools.build:gradle:2.1.0'
+  androidPlugin = 'com.android.tools.build:gradle:2.2.0'
```

Comment:
True. I only need 2.2.0 or greater for a bug fix. If it goes green I'll update and merge.